src/extra/python/isca/experiment.py

```diff
@@ -144,6 +144,8 @@ def update_namelist(self, new_vals):
     def write_namelist(self, outdir):
         namelist_file = P(outdir, 'input.nml')
         self.log.info('Writing namelist to %r' % namelist_file)
+        #A fixed column width is added to be a fixed number as most string namelist variables are width 256 in Isca, so that width plus some indentation and the namelist parameters own name should not exceed 350 characters. Default f90nml value is 72, which is regularly too short for some namelist variables where directories are pointed to.
```

Comment:
PEP8: make the comment fit 100 character width (split into several lines) and put a space after `#`

Reply:
Thanks Denis - I've now done this.